### PR TITLE
F/2.7.2

### PR DESCRIPTION
**Pull request by mhogeweg**

### CODE REVIEW

File: README.md
Comment (mhogeweg) [inline README.md:1]: f/2.7.2 file is missing the 2.7.1 release?

**Review by mhogeweg — COMMENTED** [review]
see my comment on readme.md

File: README.md
Comment (as0050629) [reply README.md:1]: Sorry, I think that was mistake. I have added 2.7.1 release

**Review by mhogeweg — COMMENTED** [review]
approved

**Pull request merged**